src/main/java/org/abstractj/kalium/keys/VerifyKey.java

```diff
@@ -41,6 +42,12 @@ public VerifyKey(String key, Encoder encoder) {
         this(encoder.decode(key));
     }
 
+    public PublicKey getPublicKey() {
+        byte []publicKey = zeros(CRYPTO_SCALARMULT_CURVE25519_BYTES);
+        sodium().crypto_sign_ed25519_pk_to_curve25519(publicKey, key);
+        return new PublicKey(publicKey);
```

Comment:
The same here, maybe we rename the method to `convertToCurve`? I believe that in the context of libsodium 25519 is implicit and we can add this to the javadocs.